spec/javascripts/PaymentGateways/stripe/components/StripeCardForm.spec.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
```

Comment:
Don't forget typing:
```suggestion
// @flow

import React from 'react'
```